src/assets/scripts/client/parsers/aircraftCommandParser/AircraftCommandParser.js

```diff
@@ -65,7 +63,7 @@ export default class AircraftCommandParser {
      * @for AircraftCommandParser
      * @param rawCommandWithArgs {string}  string present in the `$commandInput` when the user pressed `enter`
      */
-    constructor(rawCommandWithArgs = '') {
```

Comment:
is this defaulted somewhere else?  why remove the default here?

Reply:
Honestly, I'm not sure. Looked back and couldn't find any clues.... it's pretty apparent it's not _needed_, but at the same time I feel like there's no reason not to have it, so I'll add it back in.

Reply:
done